Fix customer reported issues with documentation (#1281)
cloudflare/cloudflare-docs: products/cloudflare-one/src/content/connections/connect-devices/warp/deployment/firewall.md

---
order: 10

> title: Firewall Considerations

Reviewer: remove `title`, no need to specify it

---

> 

Reviewer: ```suggestion
# WARP with Firewall
```

> If your organization utilizes a firewall or other policy to restrict internet traffic, you may need to make changes to allow WARP to connect.

Reviewer: ```suggestion
If your organization uses a firewall or other policies to restrict Internet traffic, you may need to make a few changes to allow WARP to connect.
```

## WARP Ingress IP

> The Ingress IPs are the set of IP addresses that WARP Clients will connected to. All traffic from your device to the Cloudflare edge is done via these IP addresses:

Reviewer: ```suggestion
These are the IP addresses that the WARP client will connect to. All traffic from your device to the Cloudflare edge will go through these IP addresses.
```

- IPv4 Range: 162.159.192.0/24
- IPv6 Range: 2606:4700:d0::/48

<Aside>

**Ingress IP's changing**

> In late May 2021 the ingress change for Cloudflare for Teams WARP Customers will change. The new range will be:

Reviewer: any reason we need to specify this now?

Author: My thinking is that this is a breaking change for users and they can't just walk away from this document thinking it will work forever. I am planning to work with folks to send a heads up email to all our teams customers in the next week so they have time to plan, but want to make sure I don't miss anyone.

- IPv4 Range: 162.159.193.0/24
- IPv6 Range: 2606:4700:100::/48
</Aside>

## UDP Ports

> WARP utilizes UDP for all of it's communication. UDP Port required for WARP: UDP 2408. 

Reviewer: ```suggestion
WARP utilizes UDP for all of its communications. UDP Port required for WARP: UDP 2408. 
```
